site/content/how-to/configuration/control-plane-configuration.md

```diff
@@ -1,6 +1,10 @@
-# Control Plane Configuration
-
-This document describes how to dynamically update the NGINX Gateway Fabric control plane configuration.
+---
+title: "Control Plane Configuration"
+description: "Learn how to dynamically update the NGINX Gateway Fabric control plane configuration."
+weight: 100
+toc: true
```

Comment:
is it possible to add borders to the table in this doc (and similar if any)?
Imho, it is not an easy read without borders

![image](https://github.com/nginxinc/nginx-gateway-fabric/assets/2199726/9c47f21c-d7df-4bf2-b586-68613af1b939)

Reply:
![image](https://github.com/nginxinc/nginx-gateway-fabric/assets/78599298/79b78965-a39a-4253-ab8d-072ec9c2b148)

Reply:
fixed all the tables.

We use this shortcode:
```
{{< bootstrap-table "table table-striped table-bordered" >}}

TABLE GOES HERE

{{< /boostrap-table >}}